node/src/builder.rs

```diff
@@ -0,0 +1,234 @@
+// Copyright 2020-2022 Manta Network.
+// This file is part of Manta.
```

Comment:
There seems to be some rogues commits with irrelevant code like builder.rs

Reply:
This seems like a mistake from merging manual seal mode into this so we can run it solo and someone forgot to revert it. Will clean that up.